src/test/scala/cromwell/InvalidRuntimeAttributesSpec.scala

```diff
@@ -1,50 +1,47 @@
 package cromwell
 
-import akka.event.LoggingAdapter
+import java.net.URL
+
 import akka.testkit.{EventFilter, TestActorRef}
+import cromwell.engine.ExecutionIndex._
 import cromwell.engine._
-import cromwell.engine.backend.BackendType
-import cromwell.engine.backend.jes.{JesBackend, JesBackendCall}
-import cromwell.engine.backend.runtimeattributes.CromwellRuntimeAttributes
-import cromwell.engine.workflow.{CallKey, WorkflowManagerActor}
-import cromwell.logging.WorkflowLogger
+import cromwell.engine.backend.jes.{JesAttributes, JesBackend}
+import cromwell.engine.io.gcs.{GoogleConfiguration, Refresh, ServiceAccountMode, SimpleClientSecrets}
+import cromwell.engine.workflow.WorkflowManagerActor
 import cromwell.util.SampleWdl
-import org.slf4j.LoggerFactory
-import org.specs2.mock.Mockito
-import wdl4s.{RuntimeAttributes, ThrowableWithErrors, CallInputs}
+import org.scalatest.BeforeAndAfterAll
+
+class InvalidRuntimeAttributesSpec extends CromwellTestkitSpec with BeforeAndAfterAll {
 
-import scala.concurrent.{ExecutionContext, Future}
-import scala.util.{Try, Success}
+  val testWorkflowManagerSystem = new CromwellTestkitSpec.TestWorkflowManagerSystem()
+  val actorSystem = testWorkflowManagerSystem.actorSystem
 
-class InvalidRuntimeAttributesSpec extends CromwellTestkitSpec with Mockito {
+  override protected def afterAll() = {
+    testWorkflowManagerSystem.shutdownTestActorSystem()
+    super.afterAll()
+  }
 
   "A workflow with a task with invalid runtime attributes" should {
     "fail on JES Backend" in {
-      val wd = mock[WorkflowDescriptor]
-      val logger = LoggerFactory.getLogger("Fake logger")
-      wd.workflowLogger returns logger
-      wd.shortId returns "fakeShortID"
-
-      val jesBackendCall = mock[JesBackendCall]
-      def exception = {
-        Try(CromwellRuntimeAttributes(RuntimeAttributes(Map.empty[String, Seq[String]]), BackendType.JES)).failed.get
+      val jesBackend = new JesBackend(actorSystem) {
+        private val anyString = ""
+        private val anyURL: URL = null
+        override lazy val jesConf = new JesAttributes(
+          project = anyString,
+          executionBucket = anyString,
+          endpointUrl = anyURL) {
+        }
+        override def callGcsPath(descriptor: WorkflowDescriptor, callName: String, index: ExecutionIndex): String = "gs://fake/path"
+        override def jesUserConnection(workflow: WorkflowDescriptor) = null
+        override lazy val jesCromwellInterface = null
+        override lazy val googleConf = GoogleConfiguration("appName", ServiceAccountMode("accountID", "p12"), None)
```

Comment:
Correct - this is only a fake configuration to prevent JesBackend from not initializing but we might as well be consistent anyway.